src/Mvc/Mvc.RazorPages/src/ApplicationModels/PageConventionCollection.cs

```diff
@@ -27,6 +28,13 @@ public PageConventionCollection(IList<IPageConvention> conventions)
         {
         }
 
+        internal PageConventionCollection(MvcOptions mvcOptions)
```

Comment:
There isn't a way to set the collection on the options today. Seemed like it was OK to let this be internal

Reply:
This is really ugly, but it might be the best thing we can do.